app/views/homepage/_links_and_search.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class="homepage-links-and-search">
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half govuk-grid-column-two-thirds-from-desktop homepage-links">
```

Comment:
`homepage-links` doesn't look like it's doing anything here so it can be removed.

Reply:
Good spot - removed in https://github.com/alphagov/frontend/pull/3028/commits/34273151fa0bc5d55d00653845d73cb5aa836937.